recipe/migrations/free-threaded-314.txt

```diff
@@ -0,0 +1,233 @@
+adbc-driver-manager
```

Comment:
Let's move this to `recipe/migration_support`?

Reply:
Those should also be moved. I added support for it in https://github.com/regro/cf-scripts/pull/4115/